server/Database/database.js

```diff
@@ -0,0 +1,55 @@
+require('dotenv').config();
+const mongoose = require('mongoose');
+const dbUrl = process.env.MONGO_DB_URL;
+const localHost = process.env.LOCAL_HOST_URL;
+
+class Database
+{
+    constructor()
+    {
+        const url = process.env.IN_DEV === 'true'? localHost: dbUrl;
+        const connect = mongoose.connect(url, {
+            useNewUrlParser : true,
+            useUnifiedTopology: true
+        });
+
+        connect.then((result) => {
+            console.log('Mongoose has established a connection')
+        }).catch((error) => {
+            console.log('Error --> ', error)

```

Comment:
I often run by the practice to enforce the server to crash when DB does not work.

Reply:
The thinking behind this was to help in local development. I am going to let it crash in the new changes.